sound/soc/sof/intel/hda-dai-ops.c

```diff
@@ -312,15 +312,6 @@ static int hda_ipc4_post_trigger(struct snd_sof_dev *sdev, struct snd_soc_dai *c
 		break;
 	case SNDRV_PCM_TRIGGER_SUSPEND:
 	case SNDRV_PCM_TRIGGER_STOP:
-	{
-		ret = sof_ipc4_set_pipeline_state(sdev, pipe_widget->instance_id,
-						  SOF_IPC4_PIPE_RESET);
-		if (ret < 0)
-			return ret;
-
-		pipeline->state = SOF_IPC4_PIPE_RESET;
-		break;
-	}
```

Comment:
commit message:
duplicated "So so"

also we need to explain that this follows the simplification already done for FEs, and that the DAI trigger only change the pipeline states between PAUSED and RUNNING.